### ISSUE TITLE
Any plans to modify the design for kailh pcb sockets and quick swap switches?

### ISSUE THREAD

**Issue by dmgm** [Ergodox-io/ErgoDox#5]
Most people who build a keyboard are also interested in trying different switches, and sometimes mixing switches on a board. [Kailh pcb sockets](https://novelkeys.xyz/products/kailh-pcb-sockets) seem to be a popular way to achieve this, used by the input club and ergdox ez. Are their any plans to produce a socket compatible PCB design?

**Comment by robotmaxtron** [comment]
I think that's a great idea.

**Comment by dmgm** [comment]
Other benefits are in long term maintenance, you can remove all switches to get at the board if something needs to be re-soldered or replaced. 

I've seen a kailh socket footprint file somewhere, but I doubt its as simple as cut and paste. could someone with pcb design experience comment.

**Comment by mtthwmths** [comment]
Something like what they did here? https://github.com/daprice/keyswitches.pretty

**Comment by dmgm** [comment]
Yes, thats great. If I had the skills to modify the design and hadn't bought new keyboards (with hotswap switches) since my post, I'd be right onto that.

**Comment by lingster** [comment]
hi, this looks like an interesting request that I would like to tackle. For the switch replacement is the target item: Kailh_socket_optional_reversible?

**Comment by evan-chaney** [comment]
Hi I know this is ancient, but it's something people may still be interested in. I will throw a vote towards Kailh_socket_optional_reversible being the correct one to implement. Or perhaps a hotswap and a non-hotswap version to keep PCB costs down?
